src/middlewared/middlewared/plugins/account.py

```diff
@@ -694,6 +727,7 @@ async def get_next_uid(self):
 
     @no_auth_required
     @accepts()
+    @returns(Bool('root_has_password'))
```

Comment:
It might be boring to always give names to return values, especially in cases like this when they just rephrase the method name, should we try to skip that?